broker-cli/utils/validations.js

```diff
@@ -6,6 +6,13 @@ const { Big } = require('./big')
  */
 const MAX_VALUE = '9223372036854775807'
 
+/**
+ * RegExp for matching Base64 strings, which public keys are (once stripped of newlines and banners)
+ * @constant
+ * @type {RegExp}
+ */
+const BASE64_MATCH = /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/
```

Comment:
just for reference, looks like https://stackoverflow.com/a/475217/1670474 is where the regex came from